src/cartesi-machine.lua

```diff
@@ -606,6 +649,26 @@ local options = {
             return true
         end,
     },
+    {
+        "^%-it$",
+        function(all)
+            if not all then return false end
+            htif_console_getchar = true
+            local term, lang, lc_all = os.getenv("TERM"), os.getenv("LANG"), os.getenv("LC_ALL")
+            if term then append_init = append_init .. "export TERM=" .. term .. "\n" end
+            if lang then append_init = append_init .. "export LANG=" .. lang .. "\n" end
+            if lc_all then append_init = append_init .. "export LC_ALL=" .. lc_all .. "\n" end
+            local stty <close> = assert(io.popen("stty size"))
```

Comment:
Indeed, `stty` is available in `coreutils` package on my system. In case it is not available the option will fail, and the user should either install it to use `-it` or fallback to old `-i` option.